spec/features/comments_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+RSpec.describe "Managing comments" do
+  let!(:article) { create :article }
+  let!(:user) { create :user, email: 'bob@test.com', username: 'Bob' }
+  before do
+    visit '/session/new'
+    fill_in 'Email', with: 'bob@test.com'
+    fill_in 'Password', with: 'password'
+    click_button 'Log in'
+    visit "/articles/#{article.id}"
+  end
```

Comment:
This same sign in logic is duplicated across all your specs now. It's fine for now, but before your next PR, let's refactor this to remove the duplication.